src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertFromJsonCommand.cs

```diff
@@ -92,12 +92,7 @@ protected override void EndProcessing()
                         // The first input string does not represent a complete Json Syntax. 
                         // Hence consider the the entire input as a single Json content.
                     }
-#if CORECLR
-                    catch (Newtonsoft.Json.JsonSerializationException)
```

Comment:
Since it is already handled it [here](https://github.com/2xmax/PowerShell/blob/a250f8ea5c6d3079a622007591da483832885dee/src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs#L68-L72). I decided to fix the behavior of JsonObject making them invariant of a platform rather than handling serialization exceptions for every platform. You can also take a look at [the C# unit tests](https://github.com/login/oauth/authorize?client_id=7e0a3cd836d3e544dbd9&redirect_uri=https%3A%2F%2Fgist.github.com%2Fauth%2Fgithub%2Fcallback%3Freturn_to%3Dhttps%253A%252F%252Fgist.github.com%252F2xmax%252Fced86b7f7d9bbcdf0d004fa507b9a314&response_type=code) I mentioned above.